fluentd-gcp-scaler/scaler.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
```

Comment:
Can you add an example deployment? And maybe specify what env variables are required?

Reply:
Added.